question: getting the full root module throws grpc v0.1.2 error messages

Issue running:

`go get gopkg.in/DataDog/dd-trace-go.v1/...`

Fails both locally as well as ubuntu docker image.

#14 222.0 # gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/grpc.v12
#14 222.0 root/go/pkg/mod/gopkg.in/!data!dog/dd-trace-go.v1@v1.31.0/contrib/google.golang.org/grpc.v12/grpc.go:61:11: undefined: metadata.FromContext
#14 222.0 root/go/pkg/mod/gopkg.in/!data!dog/dd-trace-go.v1@v1.31.0/contrib/google.golang.org/grpc.v12/grpc.go:92:13: undefined: metadata.FromContext
#14 222.0 root/go/pkg/mod/gopkg.in/!data!dog/dd-trace-go.v1@v1.31.0/contrib/google.golang.org/grpc.v12/grpc.go:97:9: undefined: metadata.NewContext

Dockerfile example:
```

FROM ubuntu:18.04

# Install components
RUN apt-get update && apt-get install unzip wget git curl jq iputils-ping build-essential -y

# Install golang 1.16
RUN wget https://golang.org/dl/go1.16.3.linux-amd64.tar.gz && \
    tar -xvf go1.16.3.linux-amd64.tar.gz && \
    mv go /usr/local/go-1.16

ENV GOROOT=/usr/local/go-1.16
ENV PATH=$GOROOT/bin:$PATH

RUN mkdir -p /sample_module && cd /sample_module && go get gopkg.in/DataDog/dd-trace-go.v1/...
```

@gbbr I just did some [testing](https://gist.github.com/felixge/22790a88ac9ae3a2a51862b09c04f1db)  using different versions of Go (1.14, 1.15, 1.16) and want to summarize the [results](https://drive.google.com/file/d/101CywX_tS9uJH5RVvhjMTMmlX5HKKErU/view?usp=sharing) here:

- `go get gopkg.in/DataDog/dd-trace-go.v1/...`
  - All versions print warnings about: `../go/pkg/mod/gopkg.in/!data!dog/dd-trace-go.v1@v1.31.0/contrib/google.golang.org/grpc.v12/grpc.go:61:11: undefined: metadata.FromContext`
  - Go 1.14 and Go 1.15 still update the go.mod file based on the packages that were successfully installed. [A lot of stuff](https://gist.github.com/felixge/6d09e911e0524b03ea8fccb4c3ee891b) gets added to go.mod as a result.
  - Go 1.16 no longer updates the go.mod file, so arguably the install instructions there are completely broken.
- `go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer`
  - This works consistently across all Go versions and updates the go.mod file with [minimal dependencies](https://gist.github.com/felixge/3e2ad209345a13c6255248f15832aeb2).

Based on this I think we should change the instructions in the README and anywhere else to:

```
go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
```

I'll send a PR.

I've also done some testing on the [profiler docs](https://docs.datadoghq.com/tracing/profiler/getting_started/?code-lang=go), they are currently also incorrect for all go versions. I'll fix that in the docs repo separately.

--- a/README.md
+++ b/README.md
@@ -4,11 +4,10 @@
 
 ### Installing
 
-This module contains many packages, but most users should probably install the 3 packages below:
+This module contains many packages, but most users should probably install the two packages below:
 
 ```bash
 go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
-go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentracer
 go get gopkg.in/DataDog/dd-trace-go.v1/profiler
 ```
 
@@ -27,8 +26,7 @@ Requires:
 
 ### Documentation
 
-The API is documented on [godoc](https://godoc.org/gopkg.in/DataDog/dd-trace-go.v1/ddtrace) as well as Datadog's [official documentation](https://docs.datadoghq.com/tracing/setup/go/). If you are migrating
-from an older version of the tracer (e.g. 0.6.x) you may also find the [migration document](https://github.com/DataDog/dd-trace-go/blob/v1/MIGRATING.md) we've put together helpful.
+The API is documented on [godoc](https://godoc.org/gopkg.in/DataDog/dd-trace-go.v1/ddtrace) as well as Datadog's official documentation for [Tracing Go Applications](https://docs.datadoghq.com/tracing/setup/go/) and the [Continuous Go Profiler](https://docs.datadoghq.com/tracing/profiler/getting_started/?code-lang=go). If you are migrating from an older version of the tracer (e.g. 0.6.x) you may also find the [migration document](https://github.com/DataDog/dd-trace-go/blob/v1/MIGRATING.md) we've put together helpful.
 
 ### Contributing
 

--- a/README.md
+++ b/README.md
@@ -4,9 +4,21 @@
 
 ### Installing
 
+This module contains many packages, but most users should probably install the 3 packages below:
+
 ```bash
 go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
+go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentracer
+go get gopkg.in/DataDog/dd-trace-go.v1/profiler
 ```
+
+Additionally there are many [contrib](./contrib) packages that can be installed as needed like this:
+
+```
+go get gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux
+```
+
+If you installed more packages than you intended, you can use `go mod tidy` to remove any unused packages.
 
 Requires:
 

--- a/README.md
+++ b/README.md
@@ -5,7 +5,7 @@
 ### Installing
 
 ```bash
-go get gopkg.in/DataDog/dd-trace-go.v1/...
+go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
 ```
 
 Requires:

